.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Rust
+
+on:
+  push:
+  pull_request:
+
+env:
+  WINFSP_VERSION: "2.0.23075"
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    runs-on: windows-latest
+    
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Install winfsp
+      run: choco install winfsp -y --version=${{ env.WINFSP_VERSION }}
+      
+    - name: Install winfsp-tests-x64
+      uses: engineerd/configurator@v0.0.10
+      with:
+        name: winfsp-tests-x64.exe
+        url: https://github.com/winfsp/winfsp/releases/download/v2.0/winfsp-tests-${{ env.WINFSP_VERSION }}.zip
+        pathInArchive: winfsp-tests-x64.exe
+
+    - name: Move winfsp-tests-x64
+      run: mv 'C:\\Users\\runneradmin\\runneradmin\\configurator\\bin\\winfsp-tests-x64.exe' 'C:\\Program Files (x86)\\WinFsp\\bin\\'
+
+    - name: Install rust
+      uses: actions-rust-lang/setup-rust-toolchain@f3c84ee10bf5a86e7a5d607d487bf17d57670965 # pin v1.5.0
```

Comment:
The runner already have `rust` installed, you could probably remove this step :thinking:

Reply:
This action also setup rust cache, so it's nice to keep it in fact :sweat_smile: